File: translations/codenet/gpt_commented/Go2P/gpt-4o/Code/s235491884.py
# Importing the sys module for reading input
import sys

def main():
    # Read a string from standard input
    s = sys.stdin.read().strip()  # Use strip() to remove any leading/trailing whitespace

    # Count the occurrences of the letter 'o' in the input string 's'
    n = s.count('o')

    # Calculate the result based on the count of 'o' and print it
    # The formula adds 700 to 100 times the count of 'o'
    print(700 + n * 100)

if __name__ == "__main__":
    main()

# <END-OF-CODE>

File: translations/codenet/gpt_commented/Go2C++/starcoder/Code/s235491884.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
	string s;
	cin >> s;

	// Count the occurrences of the letter 'o' in the input string's'
	int n = count(s.begin(), s.end(), 'o');

	// Calculate the result based on the count of 'o' and print it
	// The formula adds 700 to 100 times the count of 'o'
	cout << 700 + n*100 << endl;
}

